db/schema.rb

```diff
@@ -18,9 +18,9 @@
     t.text     "recipient"
     t.text     "password"
     t.text     "message_id"
-    t.boolean  "viewed"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.boolean  "authenticated"
+    t.datetime "created_at",    null: false
```

Comment:
No need to set `null: fase` [here](https://github.com/clydet/selfDestruct/blob/master/db/migrate/20151029052305_create_messages.rb#L10).  Rails will take care of timestamping for you when acting upon your records.